packages/core/src/app/payment/paymentMethod/BraintreeAchPaymentMethod.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FunctionComponent, useCallback } from 'react';
+
+import BraintreeAchPaymentForm, { BraintreeAchPaymentFormProps } from './BraintreeAchPaymentForm';
+
+type BraintreeAchPaymentMethodProps = Omit<
+    BraintreeAchPaymentFormProps,
+    'mandateText' | 'isLoadingBillingCountries' | 'usCountry' | 'initializeBillingAddressFields'
+>;
+
+
+const mandateText = 'I authorize Braintree to debit my bank account on behalf of My Online Store.';
```

Comment:
Add translation for the mandateText https://github.com/bigcommerce/checkout-js/pull/1221/files#diff-54d895cbdbf35a5073f2e2083d5da2c1ee6169b119fa629bd3e28b1d37d6cbcdR312

Reply:
Mandate text is not relevant. This text will be updated after clarification

Reply:
it's okay, but don't forget to wrap the text in translation tag